thirty-three-across/thirty-three-across-htb-validator.js

```diff
@@ -23,15 +23,39 @@ function partnerValidator(configs) {
     var result = Inspector.validate({
         type: 'object',
         properties: {
+            siteId: {
+                type: 'string',
+                minLength: 1
```

Comment:
Should we do a full validation here? That siteId should be 23 chars of [a-z][A-Z][-][0-9]?